src/software/simulation/simulator.h

```diff
@@ -176,8 +201,14 @@ class Simulator
 
     unsigned int frame_number;
 
+    // The time step used to simulate physics and primitives
+    const Duration physics_time_step;
+
     // The camera ID of all SSLDetectionFrames published by the simulator.
     // This simulates having a single camera that can see the entire field
     static constexpr unsigned int CAMERA_ID            = 0;
-    static constexpr float FIELD_LINE_THICKNESS_METRES = 0.01;
+    static constexpr float FIELD_LINE_THICKNESS_METRES = 0.01f;
+    // 200Hz is approximately how fast our robot firmware runs, so we
+    // mimic that here for physics and primitive updates
+    static constexpr double PHYSICS_TIME_STEP_SECONDS = 1.0 / 200.0;
```

Comment:
Could we rename to `DEFAULT_PHYSICS_TIME_STEP_SECONDS`?

Reply:
done